lms/templates/course_modes/track_selection_types/partial_fbe.html

```diff
@@ -0,0 +1,34 @@
+## This template is for partial Feature Based Enrollment (FBE). This means a
```

Comment:
Wondering if you need the directory `track_selection_types`? Not sure the naming is very informative, you could also leave it all under the `course_modes` directory. 
Would also consider `partial_fbe.html` to be named something else (ie. supported vs unsupported path).

Reply:
- [ ] Remove directory `track_selection_types

Task to rename `partial_fbe.html` in https://github.com/edx/edx-platform/pull/28655#discussion_r703733671.